NUModels/NUCategory.j

```diff
@@ -52,6 +61,14 @@
     return self;
 }
 
+- (id)initWithName:(CPString)aName contextIdentifier:(CPString)aContext filter:(id)aFilter
+{
+    [self initWithName:aName]
```

Comment:
This should be the contrary:
`initWithName:` should call `initWithName:contextIdentifier:filter` by setting `contextIdentifier` and `filter` to `nil`